test/InterchainTokenFactory.js

```diff
@@ -449,14 +446,31 @@ describe('InterchainTokenFactory', () => {
                         },
                     ),
                 tokenFactory,
+                'NotSupported',
+            );
+
+            await expectRevert(
+                (gasOptions) =>
+                    tokenFactory['deployRemoteInterchainToken(string,bytes32,address,string,uint256)'](
+                        '',
```

Comment:
add a test to ensure it fails if original Chain is not empty (double check test coverage for the repo, so it doesn't drop when making a PR)

Reply:
`Fails if original chain is not empty` is tested in line 449 with `chainName == "Test"`.
```
await expectRevert(
                (gasOptions) =>
                    tokenFactory['deployRemoteInterchainToken(string,bytes32,address,string,uint256)'](
                        chainName,
                        salt,
                        otherWallet.address,
                        destinationChain,
                        gasValue,
                        {
                            ...gasOptions,
                            value: gasValue,
                        },
                    ),
                tokenFactory,
                'NotSupported',
            );
```
Test coverage for `InterchainTokenFactory.sol` and `InterchainTokenService.sol ` are all 100%. Is there something I'm missing about the test case?